frontend/main-app/src/components/Footer/Footer.module.css

```diff
@@ -21,3 +22,11 @@
     color: inherit !important;
   }
 }
+
+@media screen and (max-width: 768px) {
+  .linkContainer > * {
+    text-align: center;
+    margin: auto;
+    line-height: 1.5;
+  }
```

Comment:
Try using a css variable for max-width in the media query. 

`Line-height: 1.5;` can be moved to "global" .linkContainer above, as both small and large screen sizes could benefit from more spacing 😊

You could make the links left-aligned, but reduce the size when using mobile. I think that looks more cohesive with the big screen view 😊